swayfmt/tests/mod.rs

```diff
@@ -1608,6 +1608,40 @@ impl MyContract for Contract {
     );
 }
 
+#[test]
+fn configurable_comments() {
+    check(
+        r#"
+script;
+
+use std::{constants::ZERO_B256, vm::evm::evm_address::EvmAddress};
+
+configurable {
+    // config test
+    /// config test triple
+    SIGNER: EvmAddress = EvmAddress {
+        value: ZERO_B256,
+    },
+}
+
+fn main() {}"#,
+        r#"script;
+
+use std::{constants::ZERO_B256, vm::evm::evm_address::EvmAddress};
+
+configurable {
+    // config test
+    /// config test triple
```

Comment:
Thanks for adding tests! It would be good to test inline comments as well, e.g. `/* inline test */`

Reply:
Ah, good catch! That style of comment is coming out a little strange after testing it just now. Weird behaviors with whitespace. Gonna dig into this a little more 👍

Reply:
After digging into the behaviors a little more, I discovered that this strange whitespace formatting behavior exists across other various items. I have put detailed it in this issue: #5298. Seems like what I found may be outside of the scope of this issue.

I added the `/* inline test */` comment and it is still exists after the formatting. However, the whitespace around it is a different thing. I had to form the testcase around this to make it pass.

Let me know what you think!

Reply:
Thanks for opening the issue. I agree that it's out of scope for this issue, but it would be great to have that test case added as part of #5298